app/app.go

```diff
@@ -732,7 +732,7 @@ func newETH2Client(ctx context.Context, conf Config, life *lifecycle.Manager,
 		}
 	}
 	if !ok {
-		return nil, errors.Wrap(err, "lock file fork version not in beacon node fork schedule (probably wrong chain/network)")
+		return nil, errors.New("lock file fork version not in beacon node fork schedule (probably wrong chain/network)")
```

Comment:
another occurrence of this in `dutydb/memory.go`

Reply:
Maybe double check these are the only ones?

Reply:
> Maybe double check these are the only ones?

How would you do that? Needs a linter?

Reply:
linter sure, or just ad-hoc multi-line regex?